minio/kontena.yml

```diff
@@ -0,0 +1,72 @@
+stack: kontena/minio
+version: 0.2.0
+description: Minio is a distributed object storage server built for cloud applications and devops
+variables:
+  affinity:
+    type: string
+    default: label!=no-minio
+    from:
+      prompt: Affinity
+  lb_service:
```

Comment:
Would it make sense to have the exposing through LB as optional? Just thinking that do people ALWAYS want to expose this or would there be use cases for only grid internal minio.

Reply:
But there can be an internal load balancer that isn't exposed to the public internet, right? And I suppose you'll always want to have it in front of the minio cluster.